alt_dialogs.inc

```diff
@@ -344,7 +372,7 @@ public OnPlayerClickTextDraw(playerid, Text:clickedid)
 		if(_:clickedid == 65535 || clickedid == adArrayTDs[8])
 		{
 			new tmp_dialogid = adDialogID[playerid], tmp_itemid = adItemID[playerid];
-			ShowPlayerAltDialog(playerid, AD_INVALID_ID, -1, "", "", "");
```

Comment:
btw, why omitting those arguments if the function don't allow you to skip it?
`ShowPlayerAltDialog(playerid, dialogid, style, caption[], info[], button1[], button2[] = "")`
There is only the last parameter with this option, not previous.